torchrl/data/datasets/d4rl.py

```diff
@@ -118,19 +300,23 @@ def __init__(
         split_trajs: bool = False,
         from_env: bool = True,
         use_timeout_as_done: bool = True,
+        direct_download: bool = False,
         **env_kwargs,
     ):
-
-        type(self)._import_d4rl()
-
-        if not self._has_d4rl:
-            raise ImportError("Could not import d4rl") from self.D4RL_ERR
         self.from_env = from_env
         self.use_timeout_as_done = use_timeout_as_done
-        if from_env:
-            dataset = self._get_dataset_from_env(name, env_kwargs)
+        if not direct_download:
+            type(self)._import_d4rl()
+
+            if not self._has_d4rl:
+                raise ImportError("Could not import d4rl") from self.D4RL_ERR
+
+            if from_env:
+                dataset = self._get_dataset_from_env(name, env_kwargs)
+            else:
+                dataset = self._get_dataset_direct(name, env_kwargs)
         else:
-            dataset = self._get_dataset_direct(name, env_kwargs)
+            dataset = self._get_dataset_direct_download(name, env_kwargs)
```

Comment:
I guess that `from_env` will be disabled in this case, perhaps we should check that it is not on